solutions/security/get-started/automatic-migration.md

```diff
@@ -0,0 +1,156 @@
+* **Integrations:** Shows the number of Elastic integrations that must be installed to provide data for the rule to run successfully.
+* **Actions:** Allows you to click **Install** to add a rule to Elastic. Installed rules must also be enabled before they will run. To install rules in bulk, select the check box at the top of the table before clicking **Install**.
+
+## Finalize translated rules
+
+Once you're on the **Translated rules** page, to install any rules that were partially translated or not translated, you will need to edit them. Optionally, you can also edit custom rules that were successfully translated to finetune them. 
+
+:::{note}
+You cannot edit Elastic authored rules using this interface, but after they are installed you can edit them from the [**Rules**](/solutions/security/detect-and-alert/about-detection-rules.md) page. 
+:::
+  
+### Edit a custom rule
+
+Click the rule's name to open the rule's details flyout to the **Translation** tab, which shows the source rule alongside the translated — or partially translated — Elastic version. You can update any part of the rule. When finished, click **Save**.
+
+::::{image} ../../../images/security-siem-migration-edit-rule.png
+:alt: The rule details flyout
+:width: 850px
+:screenshot:
+::::
+
+::::{note}
+If you haven't yet ingested your data, you will likely encounter `Unknown index` or `Unknown column` errors while editing. You can ignore these and add your data later.
+::::
+
+### View rule details
+
+The rule details flyout which appears when you click on a rule's name in the **Translate rules** table has two other tabs, **Overview** and **Summary**. The **Overview** tab displays information such as the rule's severity, risk score, rule type, and how frequently it runs. The **Summary** tab explains the logic behind how the rule was translated, such as why specific {{esql}} commands were used, or why a source rule was mapped to a particular Elastic authored rule.
+
+::::{important}
+All the details about your migrations is stored in the `.kibana-siem-rule-migrations-rules-default` index. You can use [Discover](/explore-analyze/discover.md) to review a variety of metrics, analyze metrics, and more.
+::::
+
+# FAQ (Frequently asked questions)
+
+**How does Automatic Migration handle rules that can't be exactly translated, such as due to feature parity issues?**
+
+After translation, rules that can't be translated appear with a status of either partially translated (yellow) or not translated (red). From there, you can address them individually.
+
+**How does Automatic Migration handle Splunk rules which lookup other indices?**
+
+Rules that fall into this category will typically appear with a status of partially translated. Lookup JOINs are currently a tech preview {{esql}} which can help in this situation. 
```

Comment:
This would be a good opportunity to link to the ES|QL lookup join  docs: https://docs-v3-preview.elastic.dev/elastic/elasticsearch/tree/main/reference/query-languages/esql/esql-lookup-join

Also, minor style change here:

```suggestion
Rules that fall into this category will typically appear with a status of partially translated. `LOOKUP JOIN`s are currently a tech preview {{esql}} which can help in this situation. 
```